tests/Doctrine/Tests/ORM/Cache/DefaultRegionTest.php

```diff
@@ -52,6 +51,23 @@ public function testSharedRegion()
         $this->assertTrue($region2->contains($key));
     }
 
+    /**
+     * @return Cache|null
+     */
+    private function createRegionCache()
+    {
+        if (extension_loaded('apc') && false !== @apc_cache_info()) {
+            return new ApcCache();
+        }
+
+        if ( ! extension_loaded('apcu')) {
```

Comment:
No idea at all, didn't want to change the behaviour... I'll check if we can do it